src/test/features/claim/form/models/claimAmountRow.ts

```diff
@@ -23,9 +23,23 @@ describe('ClaimAmountRow', () => {
     it('should deserialize all fields', () => {
       expect(ClaimAmountRow.fromObject({
         reason: 'Something',
-        amount: 100.01
+        amount: '100.01'
       })).to.deep.equal(new ClaimAmountRow('Something', 100.01))
     })
+
+    it('should deserialize amount containing comma', () => {
```

Comment:
I suggest removing these 2 tests from here. The problem of deserialising numbers does not belong to that component. I would safely assume that as long as correct deserialisation is implemented we are safe, otherwise we would need to have such tests for every place where number is asked for.